blackboxopt/utils.py

```diff
@@ -27,16 +27,29 @@ def get_loss_vector(
             of the returned loss values.
         reported_objectives: A dictionary with the objective value for each of the known
             objectives' names.
-        none_replacement: The value to use for missing objective values that are `None`
+        none_replacement: The value(s) to use for missing objective values that are
+            `None`. Either one for all objectives or a sequence for each known objective
+            respectively.
 
     Returns:
         A list of loss values.
     """
+    if isinstance(none_replacement, Sequence) and (
+        len(none_replacement) != len(known_objectives)
+    ):
+        raise ValueError(
+            f"There are only {len(none_replacement)} None replacement values but there "
```

Comment:
Nitpicking alert: This error message would also show up if `none_replacement` is longer than `known_objectives`, but the only indicates that the former is shorter :)